crates/runtime/src/lib.rs

```diff
@@ -335,7 +338,7 @@ impl<'a, T: ?Sized + Pointee> GuestPtr<'a, T> {
             .and_then(|o| self.pointer.checked_add(o));
         let offset = match offset {
             Some(o) => o,
-            None => return Err(GuestError::InvalidFlagValue("")),
+            None => return Err(GuestError::PtrOverflow),
```

Comment:
Did we really have `InvalidFlagValue` as the return error type here???

Reply:
I think that Alex wasn’t sure what error variant to use (because there wasn’t a correct one!) and I missed it in code review. No big deal

Reply:
Oh right sorry I was pretty sloppy about what errors I returned, I still need to do another quick audit pass!

Reply:
Oh, I didn't mean it as a comment about whoever left it in. Rather, I meant it as a reminder to myself that I'm getting sloppier in my code reviews.